tests/Aspire.Hosting.Tests/MongoDB/MongoDBFunctionalTests.cs

```diff
@@ -0,0 +1,31 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Aspire.Hosting.Tests.Helpers;
+using Xunit;
+
+namespace Aspire.Hosting.Tests.MongoDB;
+
+[Collection("IntegrationServices")]
+public class MongoDBFunctionalTests
+{
+    private readonly IntegrationServicesFixture _integrationServicesFixture;
+
+    public MongoDBFunctionalTests(IntegrationServicesFixture integrationServicesFixture)
+    {
+        _integrationServicesFixture = integrationServicesFixture;
+    }
+
+    [LocalOnlyFact()]
+    public async Task VerifyMongoWorks()
+    {
+        var testProgram = _integrationServicesFixture.TestProgram;
+        var client = _integrationServicesFixture.HttpClient;
+
+        using var cts = new CancellationTokenSource(TimeSpan.FromMinutes(1));
+
+        var response = await testProgram.IntegrationServiceABuilder!.HttpGetAsync(client, "http", "/mongodb/verify", cts.Token);
+
+        Assert.True(response.IsSuccessStatusCode);
```

Comment:
When the test fails, what gets logged? It might help to dump the `Results.Problem` information to the test log.

Reply:
I changed it to render the problem details as the assertion message